<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddSignalDialog</class>
 <widget class="QDialog" name="AddSignalDialog">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>628</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add signal</string>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0,0,0">
   <item>
    <layout class="QFormLayout" name="formLayout1">
     <item row="0" column="0">
      <widget class="QLabel" name="label1">
       <property name="text">
        <string>Name:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="sigNameLabel">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label2">
       <property name="text">
        <string>Number of channels:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="nChLabel">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label3">
       <property name="text">
        <string>Sampling frequency (in sps):</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="freqLabel">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="filteringGroupBox">
     <property name="toolTip">
      <string>Only for visualization, the raw signal will be saved to file</string>
     </property>
     <property name="title">
      <string>Configure filtering</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <layout class="QFormLayout" name="formLayout2">
      <item row="0" column="0">
       <widget class="QLabel" name="label4">
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="text">
         <string>Type:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="filtTypeComboBox">
        <item>
         <property name="text">
          <string>highpass</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>lowpass</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>bandpass</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>bandstop</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label5">
        <property name="text">
         <string>First critical frequency (in sps):</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="freq1TextField">
        <property name="toolTip">
         <string/>
        </property>
        <property name="placeholderText">
         <string/>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label6">
        <property name="text">
         <string>Second critical frequency (in sps):</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="freq2TextField">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="toolTip">
         <string/>
        </property>
        <property name="placeholderText">
         <string>For bandpass and bandstop only</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label7">
        <property name="text">
         <string>Filter order:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="filtOrderTextField">
        <property name="toolTip">
         <string>Order of the Butterworth filter (positive integer)</string>
        </property>
        <property name="placeholderText">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="fileSavingGroupBox">
     <property name="title">
      <string>Configure file saving</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <layout class="QFormLayout" name="formLayout3">
      <item row="1" column="1">
       <widget class="QPushButton" name="browseOutDirButton">
        <property name="text">
         <string>Browse</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label8">
        <property name="text">
         <string>Directory where the file will be saved:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label9">
        <property name="text">
         <string>Path to directory:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="outDirPathLabel">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label10">
        <property name="text">
         <string>File name:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="fileNameTextField">
        <property name="toolTip">
         <string>If empty, a name based on the timestamp will be employed</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="plotGroupBox">
     <property name="toolTip">
      <string/>
     </property>
     <property name="title">
      <string>Configure plot</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <layout class="QFormLayout" name="formLayout4">
      <item row="0" column="0">
       <widget class="QLabel" name="label11">
        <property name="text">
         <string>Channel spacing (in a.u.):</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="chSpacingTextField">
        <property name="toolTip">
         <string>Spacing between the channels in the signal unit (only for multi-channel signals)</string>
        </property>
        <property name="text">
         <string>100</string>
        </property>
        <property name="placeholderText">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label12">
        <property name="text">
         <string>Render length (in s):</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="renderLenTextField">
        <property name="text">
         <string>4</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>filteringGroupBox</tabstop>
  <tabstop>filtTypeComboBox</tabstop>
  <tabstop>freq1TextField</tabstop>
  <tabstop>freq2TextField</tabstop>
  <tabstop>filtOrderTextField</tabstop>
  <tabstop>fileSavingGroupBox</tabstop>
  <tabstop>browseOutDirButton</tabstop>
  <tabstop>fileNameTextField</tabstop>
  <tabstop>chSpacingTextField</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AddSignalDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AddSignalDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
